nss-day-cohort-30/bangazon-site-mighty-mahi-mahi
# removed the ability for a user to add a product from the home page

Fixes:
- removed the ability for a user to add a product from the home page
- 

Changes proposed in this request & reasons behind organizational or architectural decisions:
-
-

Steps to test:
- click on list of products and create new button should not show up
-
-
-

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
-
-

Commit message:
- remove create new button from index page 
Link to feature ticket:
- N/A

@mighty-mahi-mahi
